Bio/CSS/styles.css

```diff
@@ -0,0 +1,38 @@
+body {
+  background-image: url('https://pbs.twimg.com/profile_images/872735699488866304/3X7wPXrK_400x400.jpg');
+}
+
+p, a, h1, nav{
+  color: white;
+  font-weight: bolder;
+  font-family: Optima;
+  text-shadow: 8px 8px 16px black;
+  text-align:center;
+  -webkit-text-stroke: .5px black;
```

Comment:
@zdburnett when using vender prefixes always add the default rule set `text-stroke: .5px black;`

Reply:
I could not get text-stroke to work. According to this article I would need to add svg tags to all of the elements I wanted to style as such. https://stackoverflow.com/questions/38362903/why-doesnt-this-simple-html-css-text-stroke-work

I've removed text stroke altogether instead - the font actually stands out better anyway.